### ISSUE TITLE
Add The Option To Delete Character Files

### ISSUE THREAD

**Issue by EmperorJk13** [Unkzittys/DokiDokiLiteratureClub_AndroidPort#5]
You could do it in the PC, XBOX, and PS versions... 😏

**Comment by Unkzittys** [comment]
One more great suggestion! I will try to bring them and my current additions together in the 1.2.0 update. 😀

**Comment by AkiraKusanagi** [comment]
Oh yeah I forgot to add this. It's straight up impossible to get the secret/happy ending without deleting (Or being able to...) Monika's character file. So that's another reason than, 

> You could do it in the PC, XBOX, and PS versions... 😏

**Comment by AkiraKusanagi** [comment]
When this feature is implemented, maybe there should also be a directory added in the device's files called, "DDLC" so it could hold stuff like the, "Game" folder that contains the "firstrun" file (For restoring the characters upon deletion), and the "Characters" folder for the reason explained in this topic.

**Comment by Unkzittys** [comment]
Yeah, I've been wanting to do this for a while too, but I didn't know how to do it the best way. This is mainly due to the fact that the game/app folder is not deleted when it is uninstalled.

But these last few days I realized that it's kinda okay to do this by rewriting the method of operation of the firstrun file. The file will no longer be needed - instead there will be an option within the software to perform this action. It is much more practical and should make the player more sure of their action.

Lastly, I need to find a way for the user to be able to change the game directory without losing progress. Maybe I can give the choice before the game loads for the first time? I'll also need to move other standard Ren'Py files into the given folder, but I need to test how the game would behave with that first.

**Comment by AkiraKusanagi** [comment]
Understood. 👍

**Comment by AkiraKusanagi** [comment]
What if instead of going into the files and deleting characters there, you could delete them in the settings?
How it would look:

https://github.com/Unkzittys/DokiDokiLiteratureClub_AndroidPort/assets/137284132/d3374ff7-49f9-4652-b34f-f86efb8fd89f
